Makefile

```diff
@@ -156,6 +156,20 @@ release: ## Build `skywire-visor`, `skywire-cli`, `hypervisor` and apps without
 	${OPTS} go build ${BUILD_OPTS} -o ./apps/vpn-server ./cmd/apps/vpn-server
 	${OPTS} go build ${BUILD_OPTS} -o ./apps/vpn-client ./cmd/apps/vpn-client
 
+package-amd64: ## Build the debian package.
+	ln -f scripts/dPKGBUILD.sh dPKGBUILD.sh
```

Comment:
It's meant to be run from the skywire-mainnet directory, but otherwise no. I just wanted to make sure it executed in the same way as I had tested it when I moved the script into `scripts`.